docs/README.md

```diff
@@ -24,20 +24,22 @@ The following CSI interfaces are implemented:
 ### Encryption In Transit
 One of the advantages of using EFS is that it provides [encryption in transit](https://aws.amazon.com/blogs/aws/new-encryption-of-data-in-transit-for-amazon-efs/) support using TLS. Using encryption in transit, data will be encrypted during its transition over the network to the EFS service. This provides an extra layer of defence-in-depth for applications that requires strict security compliance.
 
-To enable encryption in transit, `tls` needs to be set in the `NodePublishVolumeRequest.VolumeCapability.MountVolume` object's `MountFlags` fields. For an example of using it in kubernetes, see the persistence volume manifest in [Encryption in Transit Example](../examples/kubernetes/encryption_in_transit/specs/pv.yaml)
+For EFS CSI Driver EFS CSI Driver v0.3.0 or earlier releases, this feature is optional. To enable encryption in transit, `tls` needs to be set in the `NodePublishVolumeRequest.VolumeCapability.MountVolume` object's `MountFlags` fields. For an example of using it in kubernetes, see the persistence volume manifest in [Encryption in Transit Example](../examples/kubernetes/encryption_in_transit/specs/pv.yaml)
+
+For new releases starting v0.4.0, encryption in transit is always enabled.
 
 **Note** Kubernetes version 1.13+ is required if you are using this feature in Kubernetes.
 
 ## EFS CSI Driver on Kubernetes
 The following sections are Kubernetes specific. If you are a Kubernetes user, use this for driver features, installation steps and examples.
 
 ### Kubernetes Version Compability Matrix
-| AWS EFS CSI Driver \ Kubernetes Version| maturity | v1.11 | v1.12 | v1.13 | v1.14 | v1.15 |
-|----------------------------------------|----------|-------|-------|-------|-------|-------|
-| master branch                          | beta     | no    | no    | no    | yes   | yes   |
-| v0.3.0                                 | beta     | no    | no    | no    | yes   | yes   |
-| v0.2.0                                 | beta     | no    | no    | no    | yes   | yes   |
-| v0.1.0                                 | alpha    | yes   | yes   | yes   | no    | no    |
+| AWS EFS CSI Driver \ Kubernetes Version (both master and worker node)| maturity | v1.11 | v1.12 | v1.13 | v1.14 | v1.15 |
```

Comment:
thanks. we can also just say `(worker node)` which implies the master is at least the same version because our driver does not care about master (yet). Not sure what is least confusing.